vscode-lean4/src/abbreviation/AbbreviationProvider.ts

```diff
@@ -31,6 +38,19 @@ export class AbbreviationProvider implements Disposable {
 		};
 	}
 
+	getAbbreviations() : Map<string, string> {
```

Comment:
Please just make `symbolsByAbbreviation` public instead.

Reply:
Done.

Reply:
We can remove the `getAbbreviations` and `getAbbreviationsNames` functions.

Reply:
Woops, right, removed.